index.js

```diff
@@ -6,21 +6,38 @@ const host = core.getInput('host');
 const port = core.getInput('port');
 const socketPath = core.getInput('socket-path');
 const key = core.getInput('key');
+const lifetimeInSeconds = core.getInput('lifetime');
 
 console.log(`Attempting to create ${socketPath}...`);
-try {
-    execSync(
-        'mkdir -p ~/.ssh && ' +
-        `ssh-keyscan${port ? ` -p ${port}` : ''} "${host}" >> ~/.ssh/known_hosts && ` +
-        `eval $(ssh-agent -a "${socketPath}") && ` +
-        `echo "${key}" | base64 -d | ssh-add -`
-    );
-} catch (e) {
-    console.error(e.message);
-    process.exit(1);
+
+function execSyncSafelyAndReturn(command) {
```

Comment:
This is named kind of generically for being used specifically to run ssh-agent

Reply:
Its used for two though. The output for lsof run as well as the safety of running ssh-agent.

Reply:
I guess, it's just handles a specific ssh-agent error. But this script is so small it's fine I guess.

Reply:
Oh yeah, just inlining works 👍